src/BuiltInTools/HotReloadClient/Web/BrowserConnection.cs

```diff
@@ -57,16 +68,25 @@ internal async ValueTask<bool> TrySendMessageAsync(ReadOnlyMemory<byte> messageB
         return true;
     }
 
-    internal async ValueTask<bool> TryReceiveMessageAsync(Action<ReadOnlySpan<byte>, ILogger> receiver, CancellationToken cancellationToken)
+    internal async ValueTask<bool> TryReceiveMessageAsync(ResponseAction receiver, CancellationToken cancellationToken)
     {
+#if NET
+        var writer = new System.Buffers.ArrayBufferWriter<byte>(initialCapacity: 1024);
+#else
         var writer = new ArrayBufferWriter<byte>(initialCapacity: 1024);
-
+#endif
```

Comment:
Is this #if..#else..#endif needed? The `ArrayBufferWriter<T>` in the Utilities folder is also declared in the `System.Buffers` namespace.